ui/components/ui/confusable/confusable.component.js

```diff
@@ -18,7 +18,7 @@ const Confusable = ({ input }) => {
     return (
       <Tooltip
         key={index.toString()}
-        as="span"
```

Comment:
No `as` prop exists for `Tooltip`

Reply:
@georgewrmarshall the underlying component, ReactTippy, takes tag. We may want an issue to change our Tooltip component to use 'as' but pass 'props.as' as 'tag' to ReactTippy so our surface area API for components is roughly the same.